Automattic/wp-calypso
# Help Center and FAB: Warn HEs about targetting staging

#### Proposed Changes

* This shows the below

<table>
<tr>
	<td><img width="326" alt="image" src="https://user-images.githubusercontent.com/17054134/193606547-407e7342-7f87-40d7-ac58-1a955b25c1e3.png"></td>
	<td>
<img width="421" alt="image" src="https://user-images.githubusercontent.com/17054134/193607067-cff555da-1a0d-4b44-9f77-85b8311c3d95.png">
</td>
</tr>
</table>

### When not in staging:
https://user-images.githubusercontent.com/52076348/194295108-ed4e1147-aee8-4da1-b5a5-ceb4ba6af91b.mp4

### When in staging:

https://user-images.githubusercontent.com/52076348/194295172-bb33b830-99d1-49b3-80cd-454dfb890f45.mov

#### Testing Instructions

It's kinda tricky to test this when it comes to the inline help. I'll write the instructions for the Help Center for starters.

1. Run Calypso from this branch.
2. Open the Help Center when proxied, you should get the warning.
3. Open when not proxied, you shouldn't see the warning.

Fixes: https://github.com/Automattic/wp-calypso/issues/68521
